function-server/src/main/java/io/dispatchframework/javabaseimage/Entrypoint.java

```diff
@@ -4,56 +4,32 @@
 ///////////////////////////////////////////////////////////////////////
 package io.dispatchframework.javabaseimage;
 
+import org.springframework.context.annotation.Configuration;
+
 /**
  * Main entry point for starting the function handling server
  */
 public class Entrypoint {
 
-    /**
-     * @param args
-     * @throws Exception
-     */
     public static void main(String[] args) throws Exception {
 
-        boolean springInClasspath = true;
-        try {
-            Class.forName("org.springframework.beans.factory.BeanFactory");
-        } catch (Exception e) {
-            springInClasspath = false;
-        }
+        Class c = Class.forName(args[0]);
 
-        Server server;
-        try {
-            if (springInClasspath) {
-                // run spring function server
-                server = new SpringFunctionServer(args);
-            } else {
-                // run plain old java function server
-                server = new POJFunctionServer(args[0], args[1]);
-            }
-        } catch (Exception ex) {
-            server = new ErrorServer(ex, springInClasspath);
-        }
+        Server server = (isSpringAnnotated(c) ? new SpringFunctionServer(c) : new POJFunctionServer(c));
 
         try {
+            System.err.printf("Starting Function Server for '%s'\n", args[0]);
```

Comment:
Should be System.out.printf

Reply:
I deliberately chose stderr: the message is not intended as the process output, but as a message to the user, so stderr looks like a better place.